examples/gitter-streaming.yml

```
@@ -160,5 +160,7 @@ components:
     streamingHeaders:
       http:
         headers:
-          'Transfer-Encoding': 'chunked'
-          Trailer: '\r\n'
+          type: object
+          properties:
+            'Transfer-Encoding': 'chunked'
+            Trailer: '\r\n'
```

Did you mean this instead? 🤔 

```suggestion
            'Transfer-Encoding':
              const: 'chunked'
            Trailer:
              const: '\r\n'
```